docs/content_pages/hack_pages/workshop_notes_week3.rst

```diff
@@ -13,31 +13,42 @@ Pygers Workshop Notes - Week 3 (Fall 2020)
 What is BIDS? Why do we use it?
 --------------------------------
 
-`The Brain Imaging Data Structure <https://bids.neuroimaging.io/>`_, or *BIDS*, is a **standardized way of organizing neuroimaging data**. 
+`The Brain Imaging Data Structure <https://bids.neuroimaging.io/>`_, or *BIDS*, is a **standardized way of organizing and describing neuroimaging datasets**. This helps to make data accessible and usable by different researchers over time. BIDS also standardizes metadata (i.e. the details of the task or imaging protocol) in a machine-readable way to promote automated analysis workflows, which enhances reproducibility and efficiency. When your data are in BIDS-format, you can also use BIDS apps (e.g. the BIDS validator to verify the consistency and completeness of your dataset, fMRIPrep to preprocess your MRI data, and MRIQC for data quality assurance). 
 
 ReproIn: Setting yourself up for success
 ----------------------------------------
 
+The easiest way to get started on the BIDS path is to setup your program card on the scanner (prior to data acquisition!) following the `ReproIn <https://github.com/ReproNim/reproin>`_ heuristic. This is something that will take you 30 min or less to do, and will make your life infinitely easier going forward. When you follow the ReproIn `heuristic <https://github.com/nipy/heudiconv/blob/master/heudiconv/heuristics/reproin.py>`_ on your program card, you will be able to use HeuDiConv later on to automatically convert your raw data off the scanner to BIDS format and add it to your BIDS dataset. 
```

Comment:
@emcdevitt I really like this description! Do you want to add a line or two about what to do if you're setting things up *after* data acquisition? I just feel like that might be a common question...

Reply:
Ok I added a couple sentences below!